ovos_date_parser/dates_kab.py

```
@@ -37,42 +47,330 @@
+    return False
+
+
+def _period_to_hour24(hour12: int, period_tok: str) -> int:
+    """Resolve a 1-12 spoken hour to 24h using a day-period word.
+
+    Only called when a day-period token is actually present in the
+    utterance; per the source grammar, an utterance with no period word
+    (e.g. bare "D juǧ") stays ambiguous and must default to the 12h
+    reading rather than being guessed here.
+    """
+    start, end = _DAY_PERIODS[period_tok]
```

Ruff (0.15.21)

[warning] Unpacked variable `end` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)